src/AutoRest.CSharp/Generation/Writers/LowLevelClientWriter.cs

```diff
@@ -115,7 +114,7 @@ private void WriteClientMethod(CodeWriter writer, RestClientMethod clientMethod,
 
         private void WriteClientFields(CodeWriter writer, LowLevelRestClient client, BuildContext context)
         {
-            writer.Line($"private readonly {typeof(HttpPipeline)} {PipelineField};");
+            writer.Line($"protected readonly {typeof(HttpPipeline)} {PipelineField};");
```

Comment:
Should this be a property?

Reply:
Yes, I think so. Let me give it a swing.

Reply:
Done.